src/main/kotlin/org/jitsi/videobridge/octo/OctoEndpoint.kt

```diff
@@ -95,6 +97,17 @@ class OctoEndpoint(
         return transceiver.mediaSources
     }
 
+    override fun onMaxReceiverVideoConstraintsChanged(maxVideoConstraints: VideoConstraints?) {
+        // TODO send to the specific bridge that the sender connects
+        val octoEndpointId = "[bridgeId]$id"
```

Comment:
The semantics are that this bridge wants to receive `maxVideoConstraints` for `localEndpoint`, right? Then this ID will be the same on all remote bridges, and they will replace each others' constraints.

Reply:
> The semantics are that this bridge wants to receive maxVideoConstraints for localEndpoint, right?

Yes!

> Then this ID will be the same on all remote bridges, and they will replace each others' constraints.

This is just a placeholder of the actual `bridgeId` of the local bridge, i just needed to find where to get the local bridge id.

Reply:
bb530216fd8d5ed4df642befe0c6fceb98e9ccfd addresses this